src/new-client/src/App/Trades/TradesGrid/NumFilter.tsx

```diff
@@ -0,0 +1,230 @@
+import React, { useRef } from "react"
+import { usePopUpMenu } from "utils/usePopUpMenu"
+import styled from "styled-components/macro"
+import {
+  ColField,
+  comparatorConfigs,
+  ComparatorType,
+  onColFilterEnterNum,
+} from "../TradesState"
+
+export const MultiSelectWrapper = styled.span`
```

Comment:
In this case, it's not a MultiSelect.  This is heading in the direction of becoming a generic styling component that we'll use for all three filters (number/date/set).  Don't worry about making it generic.  I could try that in my PR for the date filter.  But change the name to something like "FilterPopupWrapper".